LICENSE

```diff
@@ -83,3 +83,27 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 This Hippocratic License is an Ethical Source license
 (https://ethicalsource.dev).
+
+
+Some portions of vcr are covered by the following copyright and license notice:
```

Comment:
How can this project clearly communicate that this was for _previous_ contributions, and that current/future contributions are not covered by this license?